webui/src/Updates/UpdateLoginStatus.elm

```diff
@@ -65,20 +65,25 @@ updateLoginStatus message model =
                 )
 
         FetchVerify (Err error) ->
-            case error of
-                BadStatus response ->
-                    case response.status.code of
-                        403 ->
-                            ( { model | authRequired = Just True }
-                            , CmdUtils.delayMsg (5 * Time.second) AttemptReconnect
-                            )
+            if (not (model.wsConnected)) then
+                case error of
+                    BadStatus response ->
+                        case response.status.code of
+                            403 ->
+                                ( { model | authRequired = Just True }
+                                , CmdUtils.delayMsg (5 * Time.second) AttemptReconnect
+                                )
 
-                        _ ->
-                            ( model
-                            , CmdUtils.delayMsg (5 * Time.second) AttemptReconnect
-                            )
+                            _ ->
```

Comment:
mixed spaces and tabs

Reply:
We are using elm-format to format the code and it is also validated. WHat do you mean exactly?

Reply:
<img src="https://user-images.githubusercontent.com/918393/32415271-55a4f7b2-c237-11e7-9ae4-d1bfd6373e3b.png" height="400px" />

I thought that those darker spaces are the actual spaces mixed with tabs. Looks like Github just shows this way that the arrow is moved to the right.